simple_test.py

```diff
@@ -0,0 +1,40 @@
+import glob
+import os
+import subprocess
+
+CCDEC_BINARY_PATH = 'target/debug/examples/ccdec'
+
+def validate_decode(bitstream_path, codec):
+  with open(bitstream_path + '.md5', 'r') as golden_file:
+    golden = golden_file.read()
+  ccdec_process = subprocess.run([CCDEC_BINARY_PATH, bitstream_path, '--input-format', codec, '--output-format', 'nv12', '--compute-md5', 'frame'], capture_output=True)
+  if ccdec_process.returncode != 0:
+    print('Error running ccdec for bitstream ' + bitstream_path)
+    print(ccdec_process.stderr.decode('utf-8'))
+    return
+  test_output = ccdec_process.stdout.decode('utf-8')
```

Comment:
I don't know what's coding style recommendations for spacing between lines for Rust. 

I would personally prefer to have more spaces between lines for better readability.

Say here between L14 and L15.

Reply:
Yes, this is Python! My bad. No wonder the code look more familiar here. :) Thanks.

Reply:
I don't have good eye to catch Rust specific syntax issues say for some changes like 
- Remove anyhow from parsers
- Eliminate thiserror from parsers